docs/source/concepts/population.rst

```diff
@@ -9,8 +9,77 @@ Population Management
    :local:
    :backlinks: none
 
-Outline
--------
- - Initialization of simulants and the dependency problem.
- - Managing access with population views.
- -
+Since ``Vivarium`` is an agent-based simulation framework, managing a group of
+:term:`simulants` and their attributes is a critical task. Fundamentally, to run
+a simulation we need to be able to create new simulates, update their state
+attributes, and facilitate access to their state so that :term:`components` in
+the simulation can do interesting things based on it. The tooling to support
+working with our simulant population is called the population management system.
```

Comment:
I've been putting the intro section above the table of contents because I like the rendered formatting better.  Thoughts?